README.md

```diff
@@ -39,21 +39,13 @@ Use:
 dart format lib/ test/ -l 120
 ```
 
-In Android Studio, set the line length using Preferences -> Editor -> Code Style -> Dart -> Line length, set it to 120.  Enable auto-format with Preferences -> Languages & Frameworks -> Flutter -> Format code on save.
+In Android Studio, set the line length using Preferences → Editor → Code Style → Dart → Line length, set it to 120.  Enable auto-format with Preferences → Languages & Frameworks → Flutter → Format code on save.
 
+# Prerelease
 
-# Release
-
-Update `version` in `pubspec.yaml` to reflect this release, then
-
-## Android
+Push a git tag `v#.#.#-##`, e.g. `v0.5.2-0`, and `.github/release.yml` will build a draft release and publish it to iOS TestFlight and Android internal track.
 
-`flutter build appbundle`
 
-This will create an android app bundle at `build/app/outputs/bundle/release/`
-
-Upload the android bundle to the google play store https://play.google.com/apps/publish
-
-## iOS
+# Release
 
-In xcode, Release -> Archive then follow the directions to upload to the app store. If you have issues, https://flutter.dev/docs/deployment/ios#create-a-build-archive
\ No newline at end of file
+Update a draft release on GitHub to not contain the `-##`, after which `.github/release.yml` will build the release and publish it to the iOS App Store and Android Google Play Store.
```

Comment:
Are there additional steps that have to happen? I'm unaware.

Reply:
When you publish a draft you are removing the build number but that's not actually generating new artifacts. We are choosing the build that a human will be promoted to production in the app stores and marking it as the one we released in github.

Reply:
Updated to reflect that.